### PR TITLE
I challenged low-code with code blog article addition

### PR DESCRIPTION
**Pull request by hnybom**
Please do review the article.

### CODE REVIEW

File: _posts/2021-04-29-I-challenged-low-code-with-code.md [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+
+OutSystems offers a marketplace called Forge to share your components and libraries with other developers. Even as it's called a marketplace it's only free components. You can think of it as an npm.
+
+### Data access
+
+Data access consists of two parts, data mapping to code objects and accessing data from the database. OutSystems provides these functionalities in a standard way that is built into the platform. 
+
+OutSystems uses ORM technology as its data access. It also generates the DB schema based on the created entities in a similar way as hibernate does when hbm2ddl.auto is set to update the DB. As you know the use of hbm2ddl is discouraged in production as it can lead to unforeseen consequences with the DB schema. In my experience, this is also the case with OutSystems. If you e.g. drop a property from an entity the column will stay in the DB it's just hidden. The same is true if you delete an entity the table will still be there. This will not break the system but will lead to a bloated DB. This is undoubtedly fast but also a little risky.
```
[the remark this answers is not in the document]
Comment (mikaelpaj) [reply _posts/2021-04-29-I-challenged-low-code-with-code.md:55]: Also, the meaning of the Hibernate setting reference may not be clear to all readers?

**Review by mikaelpaj — COMMENTED** [review]
An interesting topic and a well structured post! Very clear and easy to follow, even with little previous knowledge of the techniques used.

It is also plenty long, so I didn't go through every paragraph with a fine-tooth comb! :sweat_smile:

Some overall notes:

- Automatic tools (I checked with Word quickly) find some more typos, missing commas, periods etc. I didn't note.
- At least Word also offers some stylistic suggestions that may be useful.
- Some library and tool names are not consistently capitalized.
- Could consider using more inline links, at least mid-sentence URLs.

Mostly nits in comments. As usual, not all are necessarily correct.

**Comment by hnybom** [comment]
> An interesting topic and a well structured post! Very clear and easy to follow, even with little previous knowledge of the techniques used.
> 
> It is also plenty long, so I didn't go through every paragraph with a fine-tooth comb! 
> 
> Some overall notes:
> 
> * Automatic tools (I checked with Word quickly) find some more typos, missing commas, periods etc. I didn't note.
> * At least Word also offers some stylistic suggestions that may be useful.
> * Some library and tool names are not consistently capitalized.
> * Could consider using more inline links, at least mid-sentence URLs.
> 
> Mostly nits in comments. As usual, not all are necessarily correct.

Thanks for the comments. I used https://app.grammarly.com/ for spellchecking and also for style. It might be that word has a better spellchecker.

Yeah this did come out lengthier than intended. I also threw away alot of text and ideas to keep it shorter :D Thanks for the comments.

File: _posts/2021-04-29-I-challenged-low-code-with-code.md [hunk 2]
```diff
@@ -0,0 +1,477 @@
...
+
+>  https://github.com/hnybom/low-code-vs-code
+
+## Final thoughts
+
+Here's a side by side comparison of the different aspects between OutSystems and Code
+
+| Category        | OutSystems                                                   | Code                                                         |
+| --------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| **Data access** | Automatic generation for basic access and data mapping. Also supports complex queries with SQL | Automatic generation for basic functionality, function signature-based query generation as well as support for SQL queries |
```
[the remark this answers is not in the document]
Comment (hnybom) [reply _posts/2021-04-29-I-challenged-low-code-with-code.md:462]: At least with typora it worked so it should be ok I guess.

[the remark this answers is not in the document]
Comment (hnybom) [reply _posts/2021-04-29-I-challenged-low-code-with-code.md:459]: I checked with the test docker it works.

**Review by hnybom — COMMENTED** [review]
Links and starting excerpt now work as intended

**Comment by mikaelpaj** [comment]
> Thanks for the comments. I used https://app.grammarly.com/ for spellchecking and also for style. It might be that word has a better spellchecker.

Grammarly does some things well, but even finding missing punctuation seems to be a premium feature. (Not sure if it depends on how it's used.)

This looks pretty good to me now in any case!

**Review by mikaelpaj — APPROVED** [review]

**Pull request merged**